integration-cli/docker_cli_create_test.go

```diff
@@ -321,6 +321,7 @@ func (s *DockerTrustSuite) TestTrustedIsolatedCreate(c *check.C) {
 
 	// Try create
 	icmd.RunCmd(icmd.Command(dockerBinary, "--config", "/tmp/docker-isolated-create", "create", repoName), trustedCmd).Assert(c, SuccessTagging)
+	defer os.RemoveAll("/tmp/docker-isolated-create")
```

Comment:
This should probably use a random tmp directory name instead of something hardcoded? 

This is still an improvement, so it doesn't have to be done in this PR.